src/components/layout/CompanyClaim/CompanyClaim.module.scss

```diff
@@ -1,5 +1,13 @@
 @import "../../../styles/settings.scss";
 
+$tablet-width: 768px;
+
+@mixin mobile {
```

Comment:
To nie powinno być w jakimś ogólnym pliku?

Reply:
Przeniesione do globals